tools/data_usage/console.cpp

```diff
@@ -1,109 +1,139 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#include <windows.h>
 #include <iostream>
 #include <string>
+#include <windows.h>
+#include "bpf.h"
 #include "ebpf_api.h"
+#include "libbpf.h"
 #include "ebpf_nethooks.h"
 
-#define MAP_COUNT 1
+typedef struct _app_name
+{
+    wchar_t name[32];
+} app_name_t;
 
 int main(int argc, const char** argv)
 {
     UNREFERENCED_PARAMETER(argc);
     UNREFERENCED_PARAMETER(argv);
 
-    ebpf_handle_t flow_program_handle;
-    ebpf_handle_t flow_map_handle[MAP_COUNT];
```

Comment:
Suspecting this type doesn't have a default constructor. If so, these should all be default initialized